src/atlannot/evaluation.py

```diff
@@ -0,0 +1,221 @@
+    "Thalamus": [549],
+    "Hippocampus": [1080],
+    "Cerebullum": [512],
+    "Basal Ganglia": [477, 1022, 470, 381],
+}
+
+
+def compute_jaggedness(volume, region_ids=None, axis=0):
+    """Compute the jaggedness of given region IDs for the specified volume.
+
+    Parameters
+    ----------
+    volume: np.ndarray
+        Input volume.
+    region_ids: list[int] | None
+        List of region IDs to compute the jaggedness. If None, the jaggedness
+        is computed for all the region IDs present in the volume.
+    axis: int
+        Axis along which to compute the jaggedness.
+
+    Returns
+    -------
+    results: dict[int, float]
+        Dictionary containing the region id as keys and the mean of the
+        jaggedness of that given region id as values.
+    """
+    metrics = core.compute(volume, coronal_axis_index=axis, regions=region_ids)
+    if region_ids is None:
+        return {
+            region_id: values["mean"]
+            for region_id, values in metrics["perRegion"].items()
+        }
+
+    results = {}
+    for region_id in region_ids:
+        results[region_id] = metrics["perRegion"][region_id]["mean"]
+    return results
+
+
+def compute_iou(vol_true, vol_pred, region_ids=None):
+    """Compute the intersection over union of given region IDs.
+
+    Parameters
+    ----------
+    vol_true: np.ndarray
+        Input volume considered as ground truth.
+    vol_pred: np.ndarray
+        Input volume considered as prediction.
+    region_ids: list[int] | None
+        List of region IDs to compute the intersection over union.
+        If None, the IoU is computed for all the region IDs present in the volume.
+
+    Returns
+    -------
+    results: dict[int, float]
+        Dictionary containing the region id as keys and the average of the
+        intersection over union of that given region id as values.
+    """
+    results = {}
+    if region_ids is None:
+        labels = np.unique(vol_true)
+        for label in labels:
+            results[label] = iou_score(vol_true, vol_pred, k=label)[0]
+        return results
+
+    for region_id in region_ids:
+        results[region_id] = iou_score(vol_true, vol_pred, k=region_id)[0]
+
+    return results
+
+
+def compute_region_entropy(nissl, atlas, label_value, bins=100):
+    """Compute the entropy of densities of Nissl for a given label value in the atlas.
+
+    Parameters
+    ----------
+    nissl: np.ndarray
+        Nissl volume.
+    atlas: np.ndarray
+        Annotation atlas.
+    label_value: int
+        Label Value to consider.
+    bins: int
+        Number of bins used to compute the histogram of the densities.
+
+    Returns
+    -------
+    region_entropy: float
+        Entropy of the densities of Nissl at a given region value.
+    """
+    prob_region, _ = np.histogram(nissl[atlas == label_value], bins=bins)
+    region_entropy = entropy(prob_region)
+    return region_entropy
+
+
+def compute_entropies(nissl, atlas):
```

Comment:
If we cut the computation of `brain_entropy` out of this, then maybe a better name could be `conditional_entropy`

Reply:
What do you think about this change baa8bcd ?